src/pages/ServiceDetails/ServiceInfo.tsx

```diff
@@ -139,26 +165,52 @@ class ServiceInfo extends React.Component<ServiceId, ServiceInfoState> {
             </Col>
           </Row>
           <Row className="row-cards-pf">
-            {(deployments.length > 0 || this.state.istio_sidecar) && (
-              <Col xs={12} sm={12} md={cWidth} lg={cWidth}>
-                <ServiceInfoDeployments deployments={deployments} />
-              </Col>
-            )}
-            {(dependencies.size > 0 || this.state.istio_sidecar) && (
-              <Col xs={12} sm={6} md={cWidth} lg={cWidth}>
-                <ServiceInfoRoutes dependencies={dependencies} />
-              </Col>
-            )}
-            {(routeRules.length > 0 || destinationPolicies.length > 0 || this.state.istio_sidecar) && (
-              <Col xs={12} sm={6} md={cWidth} lg={cWidth}>
-                {(routeRules.length > 0 || this.state.istio_sidecar) && (
-                  <ServiceInfoRouteRules routeRules={routeRules} />
-                )}
-                {(destinationPolicies.length > 0 || this.state.istio_sidecar) && (
-                  <ServiceInfoDestinationPolicies destinationPolicies={destinationPolicies} />
-                )}
-              </Col>
-            )}
+            <Col xs={12} sm={12} md={12} lg={12}>
+              <TabContainer id="service-tabs" defaultActiveKey={101}>
+                <div>
+                  <Nav bsClass="nav nav-tabs nav-tabs-pf">
+                    <NavItem eventKey={101}>{'Deployments (' + deployments.length + ')'}</NavItem>
```

Comment:
Can you explain how these numbers are chosen? (101, 102, 103 etc.)
In what scope do they have to be unique: in the tabcontainer?

Reply:
Let me check, I guess I used these IDs for me, as I had a inner tab and those were more clear.
But I don't remember a warning o problem, let me check.
I can change them to 1,2,3,4 if that's clear for you.